### PR TITLE
initial styling 

### PR DESCRIPTION
**Pull request by crystalgel**

### CODE REVIEW

File: views/result.ejs [hunk 1]
```diff
@@ -1,4 +1,7 @@
+<div class = "results-container">
+
 <h1><%= chosen.name %></h1><br>
+<<<<<<< HEAD
```
Comment (johnm589) [inline views/result.ejs]: Remove this

File: views/result.ejs [hunk 2]
```diff
@@ -11,8 +14,14 @@
 <% } %>
 <p><a href="<%= chosen.url %>">See Restaurant on Yelp</a></p>
 <p>phone-numbe: <%=chosen.display_phone %></p>
+=======
```
Comment (johnm589) [inline views/result.ejs]: What's this?

File: views/result.ejs [hunk 3]
```diff
@@ -23,12 +32,19 @@
...
+<<<<<<< HEAD
 <button id="shake-again">Try again</button>
 <a href="/"><button id="back">Back</button></a>
 <button id="like">GOING!</button>
 
 
+=======
+</div>
+</div>
+>>>>>>> style
```
Comment (johnm589) [inline views/result.ejs]: This

File: views/result.ejs [hunk 4]
```diff
@@ -23,12 +32,19 @@
...
+<button type="submit" id="shake-again" class="btn btn-warning btn-l">No thanks! Choose Again</button>
+<a href="/"><button class="btn btn-warning btn-l" id="back">Start Over</button></a>
 
+<<<<<<< HEAD
 <button id="shake-again">Try again</button>
 <a href="/"><button id="back">Back</button></a>
 <button id="like">GOING!</button>
 
 
+=======
```
Comment (johnm589) [inline views/result.ejs]: This

**Comment by crystalgel** [comment]
fixed

**Pull request merged**